modules/sdk-core/src/account-lib/mpc/tss/ecdsa/rangeproof.ts

```diff
@@ -258,7 +258,8 @@ export function verify(
   const modulusBytes = Math.floor((modulusBits + 7) / 8);
   const q = curve.order();
   const q3 = q ** BigInt(3);
-  if (proof.s1 == q3) {
+  // if s1 not <= q3 return false
```

Comment:
not sure we need this comment, the if statement is pretty clear

Reply:
removed